src/studio-home/tabs-section/courses-tab/courses-filters/courses-filter-menu/index.jsx

```diff
@@ -26,6 +27,13 @@ const CoursesFilterMenu = ({
     if (cleanFilters) {
       setItemMenuSelected(defaultItemSelectedText);
     }
+    if (menuType !== 'courseTypes') {
+      return;
+    }
```

Comment:
@pkulkark Why do we only make this behavior work for one type of the menu? Shouldn't default value work correctly for all menu types?